src/EditorFeatures/CSharpTest2/Recommendations/ExplicitKeywordRecommenderTests.cs

```diff
@@ -10,19 +10,19 @@
 namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests.Recommendations
 {
     [Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-    public class ExplicitKeywordRecommenderTests : KeywordRecommenderTests
+    public sealed class ExplicitKeywordRecommenderTests : KeywordRecommenderTests
     {
         [Fact]
-        public async Task TestNotAtRoot_Interactive()
+        public async Task TestAtRoot_Interactive()
```

Comment:
lots of 'implicit' and 'explicit' keyword tests changed, now that 'implict/explicit' is not just for conversion operators but for implicit/explitic extension types as well.  So they're now legal keywords in a lot more places.